### PR TITLE
Add updating homebrew instruction

### PR DESCRIPTION
**Pull request by vors**
Adding instructions for homebrew updates for maintainers

### CODE REVIEW

File: docs/maintainers/releasing.md [hunk 1]
```diff
@@ -255,3 +255,22 @@ Currently the NuGet packages published to [powershell-core feed][ps-core-feed] o
...
+========
+
+After the release, you can update homebrew formula.
+
+On macOS:
+1. Make sure that you have [homebrew cask](https://caskroom.github.io/).
+1. `brew update`
+1. `cd /usr/local/Homebrew/Library/Taps/caskroom/homebrew-cask/Casks`
+1. Edit `./powershell.rb`, reference [file history](https://github.com/vors/homebrew-cask/commits/master/Casks/powershell.rb) for the guidelines:
+1. Update `version`
```
Comment (SteveL-MSFT) [inline docs/maintainers/releasing.md]: these are supposed to be sub-bullets I think

can't these changes be scripted?

Comment (vors) [inline docs/maintainers/releasing.md]: yeah, I not sure how to format this markdown properly, so made everything flat

Comment (SteveL-MSFT) [inline docs/maintainers/releasing.md]: Put two spaces in front for sub-bullets

**Comment by daxian-dbw** [comment]
@vors A gentle reminder :) are you able to address the comments?

File: docs/maintainers/releasing.md (hunk 1 above)
Comment (vors) [inline docs/maintainers/releasing.md]: turns out 4 spaces do the trick in this case

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**